src/main/java/com/internet/shop/controllers/user/LoginController.java

```diff
@@ -18,11 +19,16 @@
     private static final Injector injector = Injector.getInstance("com.internet.shop");
     private final AuthenticationService authService =
             (AuthenticationService) injector.getInstance(AuthenticationService.class);
+    private final UserService userService = (UserService) injector.getInstance(UserService.class);
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        req.getRequestDispatcher("/WEB-INF/views/login.jsp").forward(req, resp);
+        if (userService.findByLogin("admin").isPresent()) {
+            req.getRequestDispatcher("/WEB-INF/views/login.jsp").forward(req, resp);
+        } else {
+            resp.sendRedirect(req.getContextPath() + "/inject");
+        }
```

Comment:
let's better not change this and inject admin with a button

Reply:
added